dagmc/dagnav.py

```diff
@@ -18,19 +18,30 @@ def __init__(self, moab_file):
             self.mb = core.Core()
             self.mb.load_file(moab_file)
 
+        self.used_ids = {}
+        self.used_ids[Surface._geom_dimension] = set(self.surfaces_by_id.keys())
+        self.used_ids[Volume._geom_dimension] = set(self.volumes_by_id.keys())
+        self.used_ids[Group._geom_dimension] = set(group.id for group in self.groups.values())
```

Comment:
Small thing, but using the types themselves as the keys here will insulate us from any changes to properties of the types down the line:

```suggestion
        self.used_ids[Surface] = set(self.surfaces_by_id.keys())
        self.used_ids[Volume] = set(self.volumes_by_id.keys())
        self.used_ids[Group] = set(group.id for group in self.groups.values())
```

Other changes may be required elsewhere to comply with this change.